src/MainWindow.vala

```diff
@@ -18,17 +18,14 @@
  * Authored by: Corentin Noël <corentin@elementary.io>
  */
 
-public class Mail.MainWindow : Hdy.ApplicationWindow {
+public class Mail.MainWindow : Adw.ApplicationWindow {
```

Comment:
`Adw.ApplicationWindow` prevents the window manager (or is it gtk that does it? 😅) from adding the standard title bar. I am not sure whether that's the only way to do it but I don't think `set_titlebar` would work because we've got multiple `HeaderBar`s.

Reply:
Ah okay yeah, so the way we've been dealing with this in other places is `set_titlebar (new Gtk.Grid () { visible = false });`. I'm not sure if using Adw.ApplicationWindow will have any unintended consequences on styling etc